src/main/static/src/actions/organisaatio.actions.ts

```diff
@@ -157,3 +158,15 @@ export const fetchOrganisations = (oidOrganisations: Array<string>) => (dispatch
         .then((json) => dispatch(receiveOrganisations(json)))
         .catch((e) => console.error(e));
 };
+
+export const fetchOrganisationNames = () => async (dispatch: Dispatch) => {
+    const url = urls.url('kayttooikeus-service.organisaatio.names');
+    http.get<OrganisaatioNameLookup>(url)
+        .then((payload) =>
+            dispatch({
+                type: FETCH_ORGANISATION_NAMES,
+                payload,
+            })
+        )
+        .catch((error) => console.log('Nimien lataus epäonnistui', error));
```

Comment:
Is this logging necessary and could it be implemented into some error notification if it is avail?

Reply:
It is not fatal is fetch fails, implementation shows oid:s instead. console log is there just to give a hint that something went wrong.
currently notification mechanism is such that building new notifications is quite laborsome and in this case pointless.